packages/components/src/custom-select-control/test/index.js

```diff
@@ -238,28 +248,56 @@ describe.each( [
 		).toHaveTextContent( 'Hint' );
 	} );
 
-	it( 'shows selected hint in list of options when added, regardless of __experimentalShowSelectedHint prop', async () => {
+	it( 'Should accept and pass arbitrary properties to the selectedItem object in the onChange callback', async () => {
```

Comment:
It looks like we changed an existing test. Should we keep the old test and add a new one instead?

Reply:
Added back the original test